src/elm/quantity.coffee

```diff
@@ -248,29 +242,32 @@ module.exports.parseQuantity = (str) ->
   else
     null
 
-module.exports.doAddition = (a,b) ->
+module.exports.doScaledAddition = (a,b,scaleForB) ->
```

Comment:
A little nit-picky, but... given that this is really just a utility function used internally here by add and subtract, I'd prefer not to export it -- especially since its usage isn't 100% clear outside of the context of what you're doing here.